cmd/kubeadm/app/phases/kubelet/config.go

```diff
@@ -130,26 +140,32 @@ func createConfigMapRBACRules(client clientset.Interface, k8sVersion *version.Ve
 }
 
 // DownloadConfig downloads the kubelet configuration from a ConfigMap and writes it to disk.
-// Used at "kubeadm join" time
-func DownloadConfig(client clientset.Interface, kubeletVersion *version.Version, kubeletDir string) error {
+// DEPRECATED: Do not use in new code!
```

Comment:
can we remove usage of the function after https://github.com/kubernetes/kubernetes/pull/87975 merges?

Reply:
Nope, it's still used by the `kubelet-config` phase of `kubeadm upgrade node` when `--kubelet-version` is set to a version that is different than the current one.